src/Microsoft.DotNet.ProjectModel/ProjectSystem/ProjectSystem.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Frameworks;
+
+namespace Microsoft.Extensions.ProjectModel.ProjectSystem
+{
+    public class ProjectSystem
```

Comment:
Will do. 

About the name, I want to distinguish this API from the Roslyn workspace, which do compilation additionally.